src/app/getting-started/getting-started.component.html

```diff
@@ -44,41 +44,41 @@ <h2 class="circle">2</h2>
             Please connect your GitHub and OpenShift accounts to OpenShit.io.
           </p>
           <div *ngIf="gitHubLinked && openShiftLinked; then allLinked else allUnlinked"></div>
-          <template #allLinked>
+          <ng-template #allLinked>
```

Comment:
While coding this page, I remembered we updated all templates to ng-template for Angular 4. This one slipped through with the getting started page. I can create a separate PR if you prefer?